token-exchange-server-sample/pom.xml

```diff
@@ -59,6 +59,7 @@
         </dependency>
     </dependencies>
     <properties>
+        <log4j2.version>2.15.0</log4j2.version>
```

Comment:
Please add a link to the page on the spring-boot website that explains what we are doing. Also add a hint, that we can remove the property once we updated to a fixed version of spring-boot (i.e. 2.6.2).

Reply:
Fixed